### PR TITLE
Finishes homepage styling

### PR DESCRIPTION
**Pull request by efertsch**
Complete for now until new logo is finished.

### CODE REVIEW

File: app/controllers/static_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class StaticController < ApplicationController
   def index
+    @is_mobile = browser.device.mobile?
```
Comment (jyonah) [inline app/controllers/static_controller.rb]: Love this. Can you share any materials you might have used to research this? I'm curious to learn more.

Comment (efertsch) [reply app/controllers/static_controller.rb]: I used a gem called [browser](https://github.com/fnando/browser) and it has a lot of really awesome methods for detecting browser type, devices, and platforms. Had nooo idea that this existed before today and I'm super happy about it.

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module PagesHelper
+  def is_mobile?
+    browser.device.mobile?
+  end
+end
```
Comment (jyonah) [inline app/helpers/pages_helper.rb]: Nice! I was wondering about putting this in a helper file. Love how simple the change is.

File: app/controllers/static_controller.rb (hunk 1 above)
Comment (femaleinventor) [reply app/controllers/static_controller.rb]: yep, cool

**Pull request merged**